src/Azure/Azure.Quantum.Client/Storage/IStorageHelper.cs

```diff
@@ -32,14 +32,32 @@ public interface IStorageHelper
         /// <param name="containerClient">Container client.</param>
         /// <param name="blobName">Name of the BLOB.</param>
         /// <param name="input">The input.</param>
+        /// <param name="contentType">The MIME type indicating the content of the payload.</param>
+        /// <param name="contentEncoding">The blob encoding.</param>
         /// <param name="cancellationToken">The cancellation token.</param>
         /// <returns>async task.</returns>
         Task UploadBlobAsync(
             BlobContainerClient containerClient,
             string blobName,
             Stream input,
+            string contentType,
+            string contentEncoding,
```

Comment:
```suggestion
            string? contentType,
            string? contentEncoding,
```

Reply:
Given that these two parameters can be null, it would be good to make them nullable.